src/lib.rs

```diff
@@ -7,13 +7,15 @@
 
 #[cfg(all(feature = "borsh", not(feature = "std")))]
 extern crate alloc;
+extern crate core;
```

Comment:
What does this do?

Reply:
My IDE added that. I don't think this is needed in modern Rust, so I'll remove again.